rosetta-source/src/main/rosetta/base-staticdata-asset-common-type.rosetta

```diff
@@ -8,45 +8,41 @@ import cdm.base.staticdata.party.*
 import cdm.observable.asset.*
 import cdm.product.template.*
 
-choice Asset:  <"An Asset is defined as something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the  attributes must be used."> 
+choice Asset:  <"An Asset is defined as something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the attributes must be used."> 
     Cash  <"An Asset that consists solely of a monetary holding in a currency.">
     Commodity <"An Asset comprised of raw or refined materials or agricultural products, eg gold, oil or wheat.">
     DigitalAsset <"An Asset that exists only in digital form, eg Bitcoin or Ethereum; excludes the digital representation of other Assets.">
     Instrument <"An asset that is issued by one party to one or more others; Instrument is also a choice data type.">
 
 type AssetBase: <"The base data type to specify common attributes for all Assets.">
```

Comment:
Will address in P3

Reply:
Would require addition of following enums to `AssetTypeEnum`:   DigitalAsset, Loan, ListedDerivative.  However, this enumerator is not referenced anywhere in the model.
Collateral uses `AssetClassEnum` which has values:  Commodity, Credit, Equity, ForeignExchange, InterestRate, MoneyMarket.
Reconciling this is potentially beyond scope of P3.

Reply:
`AssetType` which is used in the ECS model uses `AssetTypeEnum`, not `AssetClassEnum`.

(I agree that the latter can't be reconciled with the `Asset` model, but it doesn't need to because "asset class" usually means something different)